src/TimeGrid.js

```diff
@@ -114,7 +117,13 @@ export default class TimeGrid extends Component {
     if (this.props.width == null && !this.state.gutterWidth) {
       this.measureGutter()
     }
-
+    if (
```

Comment:
maybe stick this in `componentWillReceiveProps` instead?

Reply:
@jquense - I have made the requested change. I have also changed when the state update is made, as I realized that I was not actually solving the problem in my first commit...

State is now updated each time `triggerTimeIndicatorUpdate()` is called.

Also, I noticed another bug, but probably not a big deal.
One one minute intervals, the time Indicator will always be on the timeslot prior to the actual current minute.  I doubt many people would have this many intervals.

![image](https://user-images.githubusercontent.com/5461649/34643358-45cb30f2-f2e0-11e7-9636-7cd8840b0752.png)

Reply:
You seem to have used `componentWillMount` instead of `componentWillReceiveProps`.

Reply:
@tobiasandersen - take a look at the more recent commit here: https://github.com/intljusticemission/react-big-calendar/pull/674/commits/bec269b8aa74ca6bd3d64aebf2b03968ca19acb8

I changed the strategy a bit because I realized the componentWillReceiveProps wouldn't work.
Details of implementation:
- If user passes in the undocumented prop 'now', that is passed to the `TimeGrid` (via `{...this.props}` on `Calendar` )
    - In this case, we use `now` by setting the `now` prop on the `TimeGrid` state and keep the state up to date  with `componentWillReceiveProps`
- If `now` was not passed in, then we initialize `now` on the state to `new Date()`
- When the `triggerTimeIndicatorUpdate()` is invoked it starts the 60 second recursive timer, this will now update the `now` property on state.

Reply:
Gotcha, I was too quick and only looked through the changed code.